src/main/java/org/folio/consortia/controller/TenantController.java

```diff
@@ -31,7 +37,18 @@ public ResponseEntity<TenantCollection> getTenants(UUID consortiumId, Integer of
 
   @Override
   public ResponseEntity<Tenant> saveTenant(UUID consortiumId, @Validated Tenant tenant) {
-    return ResponseEntity.status(CREATED).body(service.save(consortiumId, tenant));
+    FolioExecutionContext currentTenantContext = (FolioExecutionContext) folioExecutionContext.getInstance();
+    // query to tenant table to get central tenant id,
+    // if not central, throw new error
+    String centralTenantId = service.getCentralTenantId(); // central tenant id is checking for null in TenantService
+    // prepare context x-okapi-tenant - tenantId == tenant.getId()
+    // call to mod-config to save
+    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(tenant.getId(), currentTenantContext, folioModuleMetadata))) {
+      configurationService.saveConfiguration(centralTenantId);
```

Comment:
https://github.com/folio-org/mod-consortia/blob/master/descriptors/ModuleDescriptor-template.json#L67

you need to add `configuration.entries.item.post` in module descriptor for POST /tenants